src/test/scala/kamon/prometheus/ScrapeDataBuilderSpec.scala

```diff
@@ -183,10 +183,27 @@ class ScrapeDataBuilderSpec extends WordSpec with Matchers {
         """.stripMargin.trim()
       }
     }
+
+    "include global custom tags from the Kamon.environment.tags" in {
+      val counterOne = MetricValue("counter-one", Map("tag.with.dots" -> "value"), time.seconds, 10)
+      val gaugeOne = MetricValue("gauge-one", Map.empty, time.seconds, 20)
+
+      builder(environmentTags = Map("env_key" -> "env_value"))
+      .appendCounters(Seq(counterOne))
+      .appendGauges(Seq(gaugeOne))
+      .build() should include {
+        """
+          |# TYPE counter_one_seconds_total counter
+          |counter_one_seconds_total{env_key="env_value",tag_with_dots="value"} 10.0
+          |# TYPE gauge_one_seconds gauge
+          |gauge_one_seconds{env_key="env_value"} 20.0
+        """.stripMargin.trim()
+      }
+    }
```

Comment:
There should be one additional test covering the environment tags vs metric tags mentioned above.